test/irb/yamatanooroti/test_rendering.rb

```diff
@@ -297,9 +297,9 @@ def test_delete
       write("continue\n")
       write("delete 0\n")
       close
-      assert_include_screen(<<~EOC)
+      assert_include_screen(<<~EOC.strip)
         (rdbg:irb) delete 0
-        deleted: #0  BP - Line  #{@ruby_file}:5 (line)
+        deleted: #0  BP - Line
```

Comment:
We need to loose this expectation because `debug` can show additional `(pending)` info based on Ruby versions (depending on iseq support).
Since we just want to test if `debug` receives the command and argument correctly, matching only this part of the message should be good enough.